baseline/lexicon.yaml

```
@@ -219,6 +254,26 @@
   synonyms:
     - Repo
     - Repositories
+- term: Secure Software Development Framework
+  definition: |
+     The NIST Secure Software Development Framework (SP 800-218) is a broadly reviewed and collaborative set of fundamental secure software development practices.
+  synonyms: 
+    - SSDF 
+    - NIST Secure Software Development Framework
+    - NIST SP 800-218
+  references: 
+    - https://csrc.nist.gov/projects/ssdf 
+    - https://csrc.nist.gov/pubs/sp/800/218/final  
+- term: Software Bill of Materials
+  definition: |
+    A manifest or list of all components that make up a given piece of software or hardware, preferably in a machine-readable/macine-parseable format.
+  synonyms: 
+    - SBOM 
+  references: 
+    - https://www.ntia.gov/sites/default/files/publications/sbom_minimum_elements_report_0.pdf
+    - https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf
+    - https://github.com/spdx
```

this should probably be
```suggestion
    - https://spdx.dev
```